component-definition.json

```diff
@@ -51,6 +51,18 @@
               }
             }
           }
+        },
+        {
+          "title": "richtext",
+          "id": "richtext",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/text/v1/rich-text",
```

Comment:
@mayur-satav I don't see we have /rich-text in core library.

Can you change this back to core/franklin/components/text/v1/text only ?

Reply:
done